### PR TITLE
Create zope.interface.travis.yml

### PR DESCRIPTION
**Pull request by genisysram**

### CODE REVIEW

**Review by gururajrkatti — CHANGES_REQUESTED** [review]
You have added "arch:" and "jobs: exclude" when there is already "jobs: include". This has completely changed the different jobs that are running in current build and your build. Pl. correct it. I think you need to add everything in existing "jobs:"

**Comment by gururajrkatti** [comment]
@genisysram any updates on this ?

**Comment by genisysram** [comment]
The Tests with **Macos** were failing, and I have raised the same in the community , they find some issue and considered for enchancements for future.
https://github.com/zopefoundation/meta/issues/63

File: travis-ymls/zope.interface.travis.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  - 3.7
+  - 3.8
+  - 3.9
+  - pypy2
+  - pypy3
+
+jobs:
+  include:
+
+    - name: Documentation
```
Comment (gururajrkatti) [inline travis-ymls/zope.interface.travis.yml:38]: I think you can include this also for ppc64le ?

File: travis-ymls/zope.interface.travis.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        - sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
+        - sphinx-build -b doctest -d docs/_build/doctrees docs docs/_build/doctest
+      after_success:
+      env: ZOPE_INTERFACE_STRICT_IRO=0
+
+    - name: CPython No C Extension
+      env: PURE_PYTHON=1
+      python: 3.8
+
+    # manylinux wheel builds
```
Comment (gururajrkatti) [inline travis-ymls/zope.interface.travis.yml:52]: I think you can include this also for ppc64le

File: travis-ymls/zope.interface.travis.yml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+   # Disable version pypy3 & pypy on ppc64le
+
+  exclude:  
+    - arch: ppc64le
+      python: pypy2
+    - arch: ppc64le
+      python: pypy3 
+    - arch: ppc64le
+      python: 2.7
```
Comment (gururajrkatti) [inline travis-ymls/zope.interface.travis.yml:115]: What is the reason for excluding this version ?

**Comment by genisysram** [comment]
@gururajrkatti 
updated all the suggested changes , Macos are failing with known issues.

File: travis-ymls/zope.interface.travis.yml [hunk 4]
```diff
@@ -70,6 +71,42 @@ jobs:
...
+    # manylinux wheel builds
+    - name: 64-bit manylinux wheels (all Pythons)
+      services: docker
+      env: DOCKER_IMAGE=quay.io/pypa/manylinux2014_ppc64le
+      install: docker pull $DOCKER_IMAGE
+      script: bash .manylinux.sh
+      arch: ppc64le
+  
+
+    - name: aarch64 wheels
```
Comment (gururajrkatti) [inline travis-ymls/zope.interface.travis.yml:102]: I think this is specific to arm64 and not needed.

**Review by gururajrkatti — COMMENTED** [review]
Pl. squash the commits

**Comment by genisysram** [comment]
@gururajrkatti 
updated as per comments

**Review by gururajrkatti — APPROVED** [review]
LGTM

**Pull request merged**